### ISSUE TITLE
[wg/math] Math Working Group Charter 2023

### ISSUE THREAD

**Issue by bert-github** [w3c/strategy#380]
New charter proposal, reviewers please take note.

# Charter Review

[Charter:](https://mathml-refresh.github.io/charter-drafts/math-2023.html)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* New
 - [ ] New WG 
 - [ ] New IG
 - If this is a new WG or IG charter request, link to [Advance Notice](), and any issue discussion: 

* Existing 
 - [x] Existing WG recharter 
 - [ ] Existing IG recharter
 - If this is a charter extension or revision, link a [diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FMath%2FDocuments%2FCharter2021.html&doc2=https%3A%2F%2Fmathml-refresh.github.io%2Fcharter-drafts%2Fmath-2023.html), and any issue discussion:

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).**

Communities suggested for outreach: a11y

Known or potential areas of concern: none

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...): this issue seems fine

Anything else we should think about as we review?

**Comment by himorin** [comment]
current running: https://www.w3.org/Math/Documents/Charter2021.html
diff: https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FMath%2FDocuments%2FCharter2021.html&doc2=https%3A%2F%2Fmathml-refresh.github.io%2Fcharter-drafts%2Fmath-2023.html

**Comment by ruoxiran** [comment]
no comment or request from APA

**Comment by himorin** [comment]
No comment nor request from i18n

**Comment by plehegar** [comment]
No comment from PING

**Comment by svgeesus** [comment]
1. Please move the "Background and Motivation" section outside the scope section, [like the charter template does](https://w3c.github.io/charter-drafts/charter-template.html). One section is informative, the other is normative and closely reviewed by lawyers.
2. The normative specifications are missing the **Adopted Draft**, **Exclusion Draft** and **Exclusion Draft charter** items. Again, refer to the [charter template](https://w3c.github.io/charter-drafts/charter-template.html#deliverables) for what to do here. Also the first specification has changed "Draft state" to "Candidate Recomentation" :)
3. The success criteria is wildly divergent from the template; it contains some interesting info on various specific specifications (which should be retained), but omits it on others. The template says what the criteria should be, and AC reps will be checking for that. Add it back, perhaps prefaced by "in general:" Also add back the clause about following the TAG Web Platform Design principles, which has been removed.
4. The patent policy section has changed "Web specifications" to the older wording, "Recommendations". Change it back; the current patent policy now gives some protections to CR, nt just to Rec as before.
5. The process document section numbers in "About this charter" are the old ones, please update them.
6. The charter history section is very incomplete, as if this proposed charter is the first ever.
7. The year is 2023, not 2020; and the copyright statement is incorrect (another consequence of starting from an old charter instead of the current template).

**Comment by plehegar** [comment]
I concur that the success criteria section needs to be aligned with the charter template. We need as much consistency between our charters as possible.

1. In its present form, the charter does not require 2 interoperable implementations for MathML Core 2 and beyond.
2. "A MathML Full (MathML 4) specification, based on Core is published." seems a low bar for success. Why list it at all?!?
3. "There is an AAM (Accessibility API Mapping) for MathML Core". This is not listed in the deliverable. Who is/will be working?

For the success criteria section, I suggest starting from the charter template and pick the items from the current charter that aren't reflected yet. If you catch yourself having to remove wording from the charter template, outline the reasons why you think the text will be harmful for the Math WG.

**Comment by plehegar** [comment]
I concur that the success criteria section needs to be aligned with the charter template. We need as much consistency between our charters as possible.

1. In its present form, the charter does not require 2 interoperable implementations for MathML Core 2 and beyond.
2. "A MathML Full (MathML 4) specification, based on Core is published." seems a low bar for success. Why list it at all?!?
3. "There is an AAM (Accessibility API Mapping) for MathML Core". This is not listed in the deliverable. Who is/will be working?

For the success criteria section, I suggest starting from the charter template and pick the items from the current charter that aren't reflected yet. If you catch yourself having to remove wording from the charter template, outline the reasons why you think the text will be harmful for the Math WG.

**Comment by plehegar** [comment]
cc @bkardell @NSoiffer

**Comment by bert-github** [comment]
On Thursday, June 29, 2023 10:02:21 PM CEST Philippe Le Hegaret wrote:
> I concur that the success criteria section needs to be aligned with
> the charter template. We need as much consistency between our
> charters as possible.
> 
> 1. In its present form, the charter does not require 2 interoperable
> implementations for MathML Core 2 and beyond.

That's on purpose. Time is too short to expect implementations of MathML 
Core 2 within this charter period, let alone of MathML Core 3.

> 2. "A MathML Full
> (MathML 4) specification, based on Core is published." seems a low
> bar for success. Why list it at all?!?

It is actually quite a bit of work. But MathML4 is a major deliverable, 
so it seems desirable that it gets a mention in the success criteria, 
even if everything was already said in the previous sections.

> 3. "There is an AAM
> (Accessibility API Mapping) for MathML Core". This is not listed in
> the deliverable. Who is/will be working?

It is a part of MathML Core 2. It is indeed not listed as a separate 
deliverable, but it is mentioned under MathML Core 2.

> 
> For the success criteria section, I suggest starting from the charter
> template and pick the items from the current charter that aren't
> reflected yet. If you catch yourself having to remove wording from
> the charter template, outline the reasons why you think the text will
> be harmful for the Math WG.

I can ask the chairs, but it seems a pity to replace a concrete list of 
success criteria with some generic text.



Bert
-- 
  Bert Bos                                ( W 3 C ) http://www.w3.org/
  http://www.w3.org/people/bos                               W3C/ERCIM
  ***@***.***                             2004 Rt des Lucioles / BP 93
  +33 (0)4 92 38 76 92            06902 Sophia Antipolis Cedex, France

**Comment by bkardell** [comment]
I noticed this morning when doing my usual (unless things are too wild) friday queue check that the charter is live and has votes and expires... *in a couple of days??*

**Comment by bert-github** [comment]
The [charter draft](https://www.w3.org/Math/Documents/Charter2023.html) has been updated:
- New success criteria section
- New time line (with dates expressed as relative dates)
- New charter history section (with the history of the previous charters)

**Comment by NSoiffer** [comment]
@bert-github: can you change the "rechartered" dates at the end from August to October. Also, the "Out of Scope" section is in bold for some reason. I think regular font is appropriate. 

I'll put this on the agenda for Thursday and see if anyone has any changes. We've already approved the charter, but I suppose it doesn't hurt to have another vote on this updated version.

**Comment by plehegar** [comment]
This has been approved for AC review. A few updates update to make before the start of the review::
1. https://github.com/w3c/charter-drafts/pull/423
2. https://github.com/w3c/charter-drafts/pull/424
3. the copyright
4. 
Group will get an extension.

**Comment by svgeesus** [comment]
Minor comment (and I see Chris Needham commented on this also) 

> It deprecates, make obsoletes, and drop features that are not used " 

is awkward English.

> It deprecates, makes obsolete, and drops features that are not used 

would be better.

**Comment by plehegar** [comment]
Announced: https://lists.w3.org/Archives/Member/w3c-ac-members/2023OctDec/0013.html
